pkg/manifests/manifests_test.go

```diff
@@ -1043,6 +1064,21 @@ ingress:
 	if p.Spec.ExternalLabels["datacenter"] != "eu-west" {
 		t.Fatal("Prometheus external labels are not configured correctly")
 	}
+
+	if p.Spec.AdditionalAlertManagerConfigs.Name != AdditionalAlertmanagerConfigSecretName {
```

Comment:
could you create a new Test function that configures AdditionalAlertmanagerConfigs and leave this function without AdditionalAlertmanagerConfigs (+ check that `p.Spec.AdditionalAlertManagerConfigs` is nil)?

Reply:
Done